cfme/tests/automate/test_method.py

```diff
@@ -749,3 +750,50 @@ def test_null_coalescing_fields(request, klass):
     )
 
     assert log.validate()
+
+
+@pytest.mark.tier(3)
+def test_automate_user_has_groups(request, appliance, custom_instance):
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1411424
+    This method should work:  groups = $evm.vmdb(:user).first.miq_groups
+    $evm.log(:info, "Displaying the user"s groups: #{groups.inspect}")
```

Comment:
**Question:** 
If possible can you please have the steps for it.? Output if any.?

Reply:
It's just simple. I just need to see ```miq_groups``` method is working or not. I don't think steps and expected result is important. hence not added.